app/controllers/request-to-go-live/organisation-address/post.controller.test.js

```diff
@@ -104,26 +107,27 @@ describe('organisation address post controller', () => {
       }
       next = sinon.spy()
       mockResponse.resetHistory()
-    })
 
-    describe('request to go live', () => {
-      it('when the required fields have not been populated, it should show the page with errors and set the ' +
-      'flags `isRequestToGoLive`, `isStripeUpdateOrgDetails`, `isSwitchingCredentials` correctly', () => {
-        const req = {
-          route: {
-            path: '/request-to-go-live/organisation-address'
-          },
-          correlationId
-        }
+      const req = {
+        ...path,
+        correlationId
+      }
 
-        next = sinon.spy()
+      next = sinon.spy()
 
-        const controller = getController(mockUpdateService)
+      const controller = getController(mockUpdateService)
 
-        controller(req, res, next)
+      controller(req, res, next)
 
-        const responseData = mockResponse.getCalls()[0]
+      responseData = mockResponse.getCalls()[0]
+    }
 
+    describe('request to go live', () => {
+      before(() => {
+        testSuiteSetup({ route: {path: '/request-to-go-live/organisation-address'}})
+      })
+
+      it('when required fields have not been populated, should display errors', () => {
```

Comment:
Can we re-order this so it reads naturally?

```suggestion
      it('should return errors when the required fields are missing', () => {
```

Reply:
Good point - done